Remove scroll-snap-type-[x/y]

queengooborg opened the pull request:
This PR removes the `scroll-snap-type-x` and `scroll-snap-type-y` CSS properties from BCD.  These properties can now be considered irrelevant as no modern browsers support either of them, as has been confirmed by the mdn-bcd-collector project.

Elchi3 reviewed (commented):
We should also remove `scroll-snap-destination` and `scroll-snap-coordinate`, right? 

I don't see a linter warning for this one, btw.

queengooborg commented:
> We should also remove `scroll-snap-destination` and `scroll-snap-coordinate`, right?

Done!  I also removed `scroll-snap-points-*` for the same reason, and confirmed Safari does not support the properties, prefixed or unprefixed.



> I don't see a linter warning for this one, btw.

The feature obsolescence linter is only enabled for the `api/` folder at this current time, otherwise the main branch would fail linting.  Once we've removed all the features that fail the linter (and confirmed they're supported or unsupported first), then we'll enable the category!

Elchi3 reviewed (approved):
Thank you! I submitted a content PR. 👍
